apps/test/unit/templates/sectionAssessments/FreeResponsesSurveyTableTest.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {mount} from 'enzyme';
+import {expect} from '../../../util/reconfiguredChai';
+import FreeResponsesSurveyTable from '@cdo/apps/templates/sectionAssessments/FreeResponsesSurveyTable';
+import {
+  surveyOne,
+  surveyTwo,
+} from '@cdo/apps/templates/sectionAssessments/assessmentsTestHelpers';
+
+describe('FreeResponsesSurveyTable', () => {
+  it('renders a table', () => {
+    const wrapper = mount(
+      <FreeResponsesSurveyTable freeResponses={surveyTwo} />
+    );
+
+    expect(wrapper.find('table')).to.exist;
```

Comment:
Is there a reason why you don't check for number of rows here (like you do in the second one)

Reply:
Oh you know, just to keep it spicy! Tbh, was matching a pattern I found in FreeResponsesAssessmentTableTest